### ISSUE TITLE
Error when using a circuit name with bang (!)

### ISSUE THREAD

**Issue by luizlopes** [VAGAScom/protoboard#11]
When I'm registering a circuit with a bang (!), like this:

```
    register_circuits({ 'create!': 'my_service/create!' },
      singleton_methods: [:create!],
      options: MY_OPTIONS
    )
```

the follow error occurs:
```
NameError:
  wrong constant name Create!CadastrosServicePerfilUsuarioCircuitProxy
# /usr/local/bundle/gems/protoboard-0.2.2/lib/protoboard/circuit_proxy_factory.rb:41:in `const_set'
# /usr/local/bundle/gems/protoboard-0.2.2/lib/protoboard/circuit_proxy_factory.rb:41:in `create_module'
# /usr/local/bundle/gems/protoboard-0.2.2/lib/protoboard/circuit_breaker.rb:86:in `create_circuit_proxy'
# /usr/local/bundle/gems/protoboard-0.2.2/lib/protoboard/circuit_breaker.rb:50:in `register_circuits'
```

thanks

**Comment by catks** [comment]
Thanks for the report.

I think this problem will also occur with other special charecters used for naming methods. (like `=`)

**Comment by catks** [comment]
Fixed in #12
